pgsqltoolsservice/language/language_service.py

```diff
@@ -142,14 +142,15 @@ def handle_completion_request(self, request_context: RequestContext, params: Tex
         def do_send_default_empty_response():
             request_context.send_response(response)
 
-        if self.should_skip_intellisense(params.text_document.uri):
-            do_send_default_empty_response()
-            return
         script_file: ScriptFile = self._workspace_service.workspace.get_file(params.text_document.uri)
         if script_file is None:
             do_send_default_empty_response()
             return
 
+        if self.should_skip_intellisense(script_file.file_uri):
+            do_send_default_empty_response()
+            return
+
         script_parse_info: ScriptParseInfo = self.get_script_parse_info(params.text_document.uri, create_if_not_exists=False)
```

Comment:
Make sure you aren't relying on the URI to be the decoded version in other places like here as well